pkg/Microsoft.Windows.Compatibility/Microsoft.Windows.Compatibility.pkgproj

```diff
@@ -27,29 +27,41 @@
     <PrereleaseLibraryPackage Include="System.Configuration.ConfigurationManager" />
     <PrereleaseLibraryPackage Include="System.Data.Odbc" />
     <PrereleaseLibraryPackage Include="System.Data.DataSetExtensions" />
-    <PrereleaseLibraryPackage Include="System.Data.SqlClient" />
-    <PrereleaseLibraryPackage Include="System.Drawing.Common" />
+    <PrereleaseLibraryPackage Include="System.Data.SqlClient">
+      <Version>4.6.1</Version>
```

Comment:
@danmosemsft @wtgodbe I moved it to 4.6.1 to react to the current servicing happening on SqlClient.